src/openrct2-ui/input/KeyboardShortcuts.cpp

```diff
@@ -299,78 +301,78 @@ ScreenCoordsXY get_keyboard_map_scroll(const uint8_t* keysState)
+    SDL_SCANCODE_H,                                    // SHORTCUT_REMOVE_BASE_LAND_TOGGLE
+    SDL_SCANCODE_V,                                    // SHORTCUT_REMOVE_VERTICAL_LAND_TOGGLE
+    SDL_SCANCODE_3,                                    // SHORTCUT_SEE_THROUGH_RIDES_TOGGLE
+    SDL_SCANCODE_4,                                    // SHORTCUT_SEE_THROUGH_SCENERY_TOGGLE
+    SDL_SCANCODE_5,                                    // SHORTCUT_INVISIBLE_SUPPORTS_TOGGLE
+    SDL_SCANCODE_6,                                    // SHORTCUT_INVISIBLE_PEOPLE_TOGGLE
+    SDL_SCANCODE_8,                                    // SHORTCUT_HEIGHT_MARKS_ON_LAND_TOGGLE
+    SDL_SCANCODE_9,                                    // SHORTCUT_HEIGHT_MARKS_ON_RIDE_TRACKS_TOGGLE
+    SDL_SCANCODE_0,                                    // SHORTCUT_HEIGHT_MARKS_ON_PATHS_TOGGLE
+    SDL_SCANCODE_F1,                                   // SHORTCUT_ADJUST_LAND
+    SDL_SCANCODE_F2,                                   // SHORTCUT_ADJUST_WATER
+    SDL_SCANCODE_F3,                                   // SHORTCUT_BUILD_SCENERY
+    SDL_SCANCODE_F4,                                   // SHORTCUT_BUILD_PATHS
+    SDL_SCANCODE_F5,                                   // SHORTCUT_BUILD_NEW_RIDE
+    SDL_SCANCODE_F,                                    // SHORTCUT_SHOW_FINANCIAL_INFORMATION
+    SDL_SCANCODE_D,                                    // SHORTCUT_SHOW_RESEARCH_INFORMATION
+    SDL_SCANCODE_R,                                    // SHORTCUT_SHOW_RIDES_LIST
+    SDL_SCANCODE_P,                                    // SHORTCUT_SHOW_PARK_INFORMATION
+    SDL_SCANCODE_G,                                    // SHORTCUT_SHOW_GUEST_LIST
+    SDL_SCANCODE_S,                                    // SHORTCUT_SHOW_STAFF_LIST
+    SDL_SCANCODE_M,                                    // SHORTCUT_SHOW_RECENT_MESSAGES
+    SDL_SCANCODE_TAB,                                  // SHORTCUT_SHOW_MAP
+    PLATFORM_MODIFIER | SDL_SCANCODE_S,                // SHORTCUT_SCREENSHOT
+    SDL_SCANCODE_MINUS,                                // SHORTCUT_REDUCE_GAME_SPEED,
+    SDL_SCANCODE_EQUALS,                               // SHORTCUT_INCREASE_GAME_SPEED,
+    PLATFORM_MODIFIER | ALT | SDL_SCANCODE_C,          // SHORTCUT_OPEN_CHEAT_WINDOW,
+    SDL_SCANCODE_T,                                    // SHORTCUT_REMOVE_TOP_BOTTOM_TOOLBAR_TOGGLE,
+    SDL_SCANCODE_UP,                                   // SHORTCUT_SCROLL_MAP_UP
+    SDL_SCANCODE_LEFT,                                 // SHORTCUT_SCROLL_MAP_LEFT
+    SDL_SCANCODE_DOWN,                                 // SHORTCUT_SCROLL_MAP_DOWN
+    SDL_SCANCODE_RIGHT,                                // SHORTCUT_SCROLL_MAP_RIGHT
+    SDL_SCANCODE_C,                                    // SHORTCUT_OPEN_CHAT_WINDOW
+    PLATFORM_MODIFIER | SDL_SCANCODE_F10,              // SHORTCUT_QUICK_SAVE_GAME
+    static_cast<uint16_t>(Input::Shortcut::Undefined), // SHORTCUT_SHOW_OPTIONS
```

Comment:
this is not Input::Shortcut::Undefined but a different variable with the same value.